paket.lock

```diff
@@ -1,42 +1,46 @@
-FRAMEWORK: NET40, NET45
+FRAMEWORK: NET40, NET45, NET452
 NUGET
   remote: https://www.nuget.org/api/v2
-    LibGit2Sharp (0.22)
-      LibGit2Sharp.NativeBinaries (1.0.129)
-    LibGit2Sharp.NativeBinaries (1.0.129)
+    LibGit2Sharp (0.23.1)
+      LibGit2Sharp.NativeBinaries (1.0.164)
+    LibGit2Sharp.NativeBinaries (1.0.164)
     MSBuildTasks (1.4.0.45)
-    NLog (4.3.10)
-    NuGet.CommandLine (3.4.3)
+    NLog (4.4.5)
+    NuGet.CommandLine (3.5)
     WixSharp (1.0.22.3)
-    xunit (2.1)
-      xunit.assert (2.1)
-      xunit.core (2.1)
-    xunit.abstractions (2.0) - framework: net45
-    xunit.assert (2.1)
-    xunit.core (2.1)
-      xunit.extensibility.core (2.1) - framework: net45
-      xunit.extensibility.execution (2.1) - framework: net45
-    xunit.extensibility.core (2.1) - framework: net45
-      xunit.abstractions (2.0)
-    xunit.extensibility.execution (2.1) - framework: net45
-      xunit.extensibility.core (2.1) - framework: net45
-    xunit.runner.console (2.1)
-    xunit.runner.msbuild (2.1)
-    xunit.runner.visualstudio (2.1)
+    xunit (2.2)
+      xunit.assert (2.2)
+      xunit.core (2.2)
+    xunit.abstractions (2.0.1) - framework: net452
+    xunit.assert (2.2)
+    xunit.core (2.2)
```

Comment:
whoa, was it intentional to upgrade all the packages in this commit? Do we need them for something?